pkgs/tools/filesystems/xfsdump/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchgit
+, attr
+, gettext
+, autoconf
+, automake
+, ncurses
+, libtool
+, libuuid
+, libxfs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfsdump";
+  version = "3.1.10";
+
+  src = fetchgit {
+    url = "git://git.kernel.org/pub/scm/fs/xfs/xfsdump-dev.git";
+    rev = "3607469d0a8ca3513bfb6093a0da9932dddc8979";
+    sha256 = "sha256-rvclxQUJ80J0+RuwjyK3LyTvSZnDmzvmgboAPFsXFYM=";
+  };
+
+  nativeBuildInputs = [
+    autoconf automake gettext libtool 
+  ];
+  buildInputs = [
+    attr libuuid libxfs ncurses
+  ];
+
+  patchPhase = ''
+    substituteInPlace Makefile \
+      --replace "cp include/install-sh ." "cp -f include/install-sh ."
+  '';
+
+  preConfigure = ''
+    export MAKE=$(type -P make)
+    export MSGFMT=$(type -P msgfmt)
+    export MSGMERGE=$(type -P msgmerge)
+    export XGETTEXT=$(type -P xgettext)
+
+    make configure
+    patchShebangs ./install-sh
+  '';
+
+  meta = with lib; {
+    description = "XFS filesystem incremental dump utility";
+    homepage = "https://www.gnu.org/software/hello/manual/";
+    changelog = "http://xfs.org";
+    license = licenses.gpl2;
```

Comment:
People have been trying to transition away from this to stuff like `gpl2Plus` or `gpl2Only`, because `gpl2` is (legally) ambiguous.